### DISCUSSION TITLE
Issues with creating new team - slug name already in use

### DISCUSSION THREAD

**Discussion by cesconix** [vercel/community#2602] [Help] [answered]
### Summary

I have been attempting to create a new team on Vecel under a specific name. However, it seems that another account is already using the slug name. Previously, I created a team with the same name from a different account, which I have since deleted.

I expected that upon deleting the old team, the slug name would be available for use once again, but this does not appear to be the case. As such, I cannot create my new team under the desired slug name.

Could you please assist in resolving this issue? If it is necessary to provide further details or confirm previous account information for verification purposes, I am more than willing to cooperate.

Thank you in advance for your time. I look forward to hearing from you soon.

### Steps to Reproduce

Go to Settings --> Teams --> Create new Team

**Comment by amyegan** [comment, marked as answer]
Hi @cesconix! If you can share the original team name/slug then I can get some more info on what happened to it.

**Reply by cesconix** [reply]
Hi @amyegan, the team name is Enabling Solutions. Thanks

**Reply by amyegan** [reply]
It looks like there is a team with that name currently associated with your account (I'm basing this on your GitHub username linked to the account). The team was on a Pro Trial that expired, but you shouldn't need to create a new team. All you need to do is upgrade to a Pro plan to resume using advanced features.

**Reply by cesconix** [reply]
Ok, thanks for the analysis. 🙏

I need to move the team ownership from my personal Vercel account (with email cesconix@me.com) to the work Vercel account (with email francesco.pasqua@jakala.com).

Is it possible?

**Reply by amyegan** [reply]
I can extend the trial for a couple of days to let you transfer ownership to your work account. Let me know if that sounds like an acceptable solution and I will update the trial

**Reply by cesconix** [reply]
Right now, I don't see the Team "Enabling Solutions" on my personal Vercel account because I deleted it some days ago. Can you confirm that I will see the team when you will extend the trial for a couple of days? If yes, you can proceed, thanks!

**Reply by cesconix** [reply]
Hey @amyegan! Everything is ok now! I found the team, and I moved it to my work account. Thanks.
